patterns/atoms/global/00-colors.mustache

```diff
@@ -0,0 +1,38 @@
+<ul class="sg-colors">
+	<li>
+		<span class="sg-swatch" style="background: #ff0000;"></span>
+		<span class="sg-label">#ff0000</span>
```

Comment:
@rebmullin, good point, I'm ok with 2 or 4 spaces for html, 2 for JS and CSS/SASS. But my opinion that we should avoid tabs, it makes code reviews harder